packages/cmf/src/cmfConnect.js

```diff
@@ -169,6 +169,11 @@ export default function cmfConnect({
 					this.props,
 					{ dispatchActionCreator: this.dispatchActionCreator },
 				);
+				// remove all internal props already used by the container
```

Comment:
I would suggest to set a `CMF_PROPS` array + a function that delete all those props.
So adding a new cmf props will automatically remove it here. Maintaining an array of const is easier than maintaining code